README.md

```diff
@@ -1,4 +1,47 @@
 metro
 =====
 
-Module for sending metric to configurable pipelines
+Node module for sending metrics to graphite time series DB
+
+How to use metro in your repo:
+==============================
+
+1. `npm install metro -s`
+
+2. As part of the setup process create factory for graphite pipeline. Recommended to do it in a dedicated file. Example:
+
+    const config = require('config');
+    const metro = require('metro');
+
+    module.exports = function setupMetro() {
+
+      let metroClient = null;
+      metro.create('default');
+      if (config.graphite.host) {
+        metroClient = metro.get('default');
+        metroClient.addPipeline(metro.pipelineCreator('graphite', config.graphite.prefix, {
+          host: config.graphite.host,
+          port: config.graphite.port
+        }));
+      }
+
+      return metroClient;
+    };
+
+3. init metro in your app.js, example:
+
+    const metro = require('setup/metro');
```

Comment:
I'd remove step 3 and 4, and simply add the usage in the same file that configured metro for simplicity.

Bootleg is not an industry standard, and it might confuse people that just wants to use metro as is.

Reply:
I think its depend if its specific for bigpanda or more general instructions for open source community.

Reply:
Opensource